src/main/resources/i18n/Enums_es.properties

```diff
@@ -0,0 +1,141 @@
+# GENERATED Wed, 15 Mar 2023 17:03:17 GMT - DO NOT EDIT
+nursery.BatchQuantityHistoryType.Computed=Computado
+nursery.BatchQuantityHistoryType.Observed=Observado
+nursery.WithdrawalPurpose.Dead=Muerto
+nursery.WithdrawalPurpose.NurseryTransfer=Transferencia de viveros
+nursery.WithdrawalPurpose.Other=Otros
+nursery.WithdrawalPurpose.OutPlant=Planta exterior
```

Comment:
I think this one is wrong, I would use "Transplantar" or "Transplante", because "planta exterior" means "exterior plant"